feat: Add Systemd management

ANGJustinl opened the pull request:
添加快捷配置systemd的单行命令启动方式

ANGJustinl committed:
feat: Add Systemd management

添加快速启动守护进程教程

Muska-Ami commented on Writerside/topics/使用-Systemd-管理和守护进程.md:
```diff
@@ -1,5 +1,55 @@
+请记住后续操作中用到的 Unit 名称 是 lcf-frpc@<启动参数>，例如 lcf-frpc@你的token:隧道号
+
+## 启动并设置自动启动
+例如你的token是1919810 想要启动的隧道号是114514则:
```
建议使用更规范的表达

Muska-Ami commented on Writerside/topics/使用-Systemd-管理和守护进程.md:
````diff
@@ -1,5 +1,55 @@
+WantedBy=multi-user.target
+```
+
+请记住后续操作中用到的 Unit 名称 是 lcf-frpc@<启动参数>，例如 lcf-frpc@你的token:隧道号
````
可以使用 markdown 代码语法或代码块以实现更好的示例

Muska-Ami reviewed (changes requested):
Need changes

ANGJustinl committed:
feat: 使用更规范的表达

Updated the documentation for managing and running Frpc with Systemd, including two methods for configuration and service setup.

ANGJustinl commented:
> Need changes

已使用更规范的表达修改.

ANGJustinl committed:
feat: 更新 Docs tree，增加新方案和自定义配置文件支持

ANGJustinl reviewed (commented):
已修改, 不过不知道对不对 🥲

Muska-Ami commented:
迁移到新的文档系统了，不知道你是否愿意更新一下？新仓库在 [LoCyan-Team/LoCyanFrpDocs](https://github.com/LoCyan-Team/LoCyanFrpDocs)
